web/src/server/api/services/EventProcessor.ts

```diff
@@ -520,52 +519,53 @@ export class ScoreProcessor implements EventProcessor {
   ): Promise<Trace | Observation | Score> {
     const { body } = this.event;
 
-    const accessCheck = await checkApiAccessScope(
-      apiScope,
-      [
-        { type: "trace", id: body.traceId },
-        ...(body.observationId
-          ? [{ type: "observation" as const, id: body.observationId }]
-          : []),
-      ],
-      "score",
-    );
-    if (!accessCheck)
-      throw new ForbiddenError("Access denied for score creation");
+    if (apiScope.accessLevel !== "scores" && apiScope.accessLevel !== "all")
```

Comment:
apiScope.accessLevel !== "all" will fail requests from the web sdk, or not?

Reply:
this is && meaning if either "scores" or "all" is the access level, this will not fail